person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name, parent_permission = `true`)
```

Comment:
- [ ] **[OPTIONAL]** This is beyond the scope of this project so I am making it optional, but I think you should check it out. I was unable to create a person due to this error (see image). Considering that name should be optional and `parent_permission` should be `true` (boolean), not `'true'` (string), I think you should fix your constructor parameters.

  ```
  def initialize(age, name = 'Unknown', parent_permission: true)
  ```

  ![image](https://user-images.githubusercontent.com/36542986/225251969-67a87e0f-aa79-43c4-af96-291bbce2acbb.png)